src/SSHDebugPS/SSH/SSHConnection.cs

```diff
@@ -86,12 +105,17 @@ internal void CopyFile(string sourcePath, string destinationPath)
             _remoteSystem.FileSystem.UploadFile(sourcePath, destinationPath);
         }
 
+        public override void CopyDirectory(string sourcePath, string destinationPath)
+        {
+            throw new NotImplementedException();
```

Comment:
>throw new NotImplementedException(); [](start = 12, length = 36)

Why is this not implemented?  I believe `IRemoteFileSystem.UploadDirectory` will do it.